packages/salesforcedx-utils-vscode/src/helpers/utils.ts

```diff
@@ -14,9 +14,12 @@ export const isNullOrUndefined = (object: any): object is null | undefined => {
 };
 
 export const extractJsonObject = (str: string): any => {
-  const jsonString = str.substring(str.indexOf('{'), str.lastIndexOf('}') + 1);
+  const isPlainText = str.includes('{') ? false : true;
```

Comment:
@CristiCanizales why is this change necessary?

Reply:
The original code's intent was to return a parsed json string.

As you state, this function is being called with an invalid json string.

While this change avoids parsing an invalid json string, it also changes the intent of the function.

The risk here is returning a string to a calling function that is expecting an object. If the caller is assuming an object and tries to access an expected property on an object and this function returns a string, the access will fail.

Have a look at alternatives on how to handle an invalid json string.